src/i18n/es.js

```diff
@@ -167,7 +167,7 @@ export default {
       </p>
       `,
     domain: `Dominio`,
-    domain_not_verified: `Sin visitas registradas`,
+    domain_not_verified: `Sin visitas registradas [Verificar dominio](${urlSiteTracking}).`,
```

Comment:
Remember the suffix.

```suggestion
    domain_not_verified_MD: `Sin visitas registradas [Verificar dominio](${urlSiteTracking}).`,
```

Reply:
fixed